src/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixer.php

```diff
@@ -38,10 +38,12 @@
  *
  * @phpstan-type _AutogeneratedInputConfiguration array{
  *  call_type?: 'self'|'static'|'this',
+ *  call_type_for_former?: 'inherit'|'none'|'this',
```

Comment:
what about introducing the `target` option, like other `PhpUnit***Fixer`, and base the type of call based on version?